examples/IBLevelSet/ex1/input2d

```diff
@@ -0,0 +1,560 @@
+   visit_number_procs_per_file = 1
+
+// restart dump parameters
+   restart_interval      = 0                 // zero to turn off
+   restart_write_dirname = "restart_IB2d"
+
+// hierarchy data dump parameters
+   hier_dump_interval = 0                    // zero to turn off
+   hier_dump_dirname  = "hier_data_IB2d"
+
+// timer dump parameters
+   timer_dump_interval   = 0                 // zero to turn off
+
+// post processor parameters
+   postprocess_interval = 0                  // zero to turn off
+}
+
+CartesianGeometry {
+   domain_boxes = [ (0,0) , (Nx - 1 , Ny - 1) ]
+   x_lo         = -LENGTH, -HEIGHT         // lower end of computational domain.
+   x_up         =  LENGTH, HEIGHT          // upper end of computational domain.
+   periodic_dimension = 1, 1
+}
+
+GriddingAlgorithm {
+   allow_patches_smaller_than_minimum_size_to_prevent_overlaps = TRUE
+   coalesce_boxes = TRUE
+
+   max_levels = MAX_LEVELS           // Maximum number of levels in hierarchy.
+   ratio_to_coarser {
+      level_1 = REF_RATIO,REF_RATIO  // vector ratio to next coarser level
+      level_2 = REF_RATIO, REF_RATIO
+      level_3 = REF_RATIO_FINEST,REF_RATIO_FINEST
+      level_4 = REF_RATIO_FINEST,REF_RATIO_FINEST
+      level_5 = REF_RATIO_FINEST,REF_RATIO_FINEST
+   }
+
+   largest_patch_size {
+      level_0 =512,512  // largest patch allowed in hierarchy
+                        // all finer levels will use same values as level_0...
+   }
+
+   smallest_patch_size {
+      level_0 = 8, 8 // smallest patch allowed in hierarchy
+                     // all finer levels will use same values as level_0...
+   }
+
+   efficiency_tolerance   = 0.85e0    // min % of tag cells in new patch level
+   combine_efficiency     = 0.85e0    // chop box if sum of volumes of smaller
+                                      // boxes < efficiency * vol of large box
+}
+
+Nx_lower1 = int( (LENGTH/2 - 3.6)/H_COARSEST )
+Ny_lower1 = int( (HEIGHT/2 - 2.5)/H_COARSEST )
+
+Nx_upper1 = int( (LENGTH/2 + 3.4)/H_COARSEST )
+Ny_upper1 = int( (HEIGHT/2 + 2.5)/H_COARSEST )
+
+Nx_lower2 = int( (LENGTH/2 - 2.8)/H_COARSEST )
+Ny_lower2 = int( (HEIGHT/2 - 1.7)/H_COARSEST )
+
+Nx_upper2 = int( (LENGTH/2 + 2.7)/H_COARSEST )
+Ny_upper2 = int( (HEIGHT/2 + 1.7)/H_COARSEST )
+
+Nx_lower3 = int( (LENGTH/2 - 2.3)/H_COARSEST )
+Ny_lower3 = int( (HEIGHT/2 - 1.4)/H_COARSEST )
+
+Nx_upper3 = int( (LENGTH/2 + 2.3)/H_COARSEST )
+Ny_upper3 = int( (HEIGHT/2 + 1.4)/H_COARSEST )
+
+Nx_lower4 = int( (LENGTH/2 - 1.8)/H_COARSEST )
+Ny_lower4 = int( (HEIGHT/2 - 1.0)/H_COARSEST )
+
+Nx_upper4 = int( (LENGTH/2 + 1.8)/H_COARSEST )
+Ny_upper4 = int( (HEIGHT/2 + 1.0)/H_COARSEST )
+
+Nx_lower5 = int( (LENGTH/2 - 1.2)/H_COARSEST )
+Ny_lower5 = int( (HEIGHT/2 - 0.6)/H_COARSEST )
+
+Nx_upper5 = int( (LENGTH/2 + 1.2)/H_COARSEST )
+Ny_upper5 = int( (HEIGHT/2 + 0.5)/H_COARSEST )
+
+StandardTagAndInitialize {
+   //tagging_method = "GRADIENT_DETECTOR"
+   tagging_method = "REFINE_BOXES"
```

Comment:
Add one input file with "GRADIENT DETECTOR" also. In that case, turn off vorticity tagging. The example I had given you for falling cylinder on air-water interface has gradient detector. It will tag cells based on solid level set. Take a look.